package.json

```diff
@@ -25,5 +25,8 @@
         "@firebase/app-types"
       ]
     }
+  },
+  "dependencies": {
+    "firebase": "^10.11.1"
```

Comment:
I think this was an accident. Might mean root-level `package-lock.json` update was too.